examples/jsm/loaders/3DMLoader.js

```diff
@@ -121,12 +122,22 @@ class Rhino3dmLoader extends Loader {
 
 					worker.postMessage( { type: 'decode', id: taskID, buffer }, [ buffer ] );
 
-					//this.debug();
+					// this.debug();
 
 				} );
 
 			} )
-			.then( ( message ) => this._createGeometry( message.data ) );
+			.then( ( message ) => {
+
+				const geometry = this._createGeometry( message.data );
```

Comment:
@karimi I don't think this is needed anymore since there aren't warnings attached to geometry.